content/en/case-studies/blackhole-image.md

```diff
@@ -0,0 +1,62 @@
+
+### Key Imaging Objectives
+
+* To study the most extreme objects in the Universe predicted by Einstein’s theory of general relativity, during the centennial year of the historic experiment that first confirmed the theory (2017).
+
+* Focus on the [black hole](https://solarsystem.nasa.gov/resources/2319/first-image-of-a-black-hole/) at the center of Messier 87 galaxy, located in the Virgo galaxy cluster. This black hole resides approximately 55 million light-years from Earth and has a mass equal  to 6.5 billion times that of the Sun. It has been a subject of astronomical study for [over a 100 years](https://www.jpl.nasa.gov/news/news.php?feature=7385). Black holes have been theoretically predicted and observed but a real image was never created until now.
+
+* Based on Einstein’s general theory of relativity, the scientists expected to see a dark region similar to a shadow, caused by the gravitational bending and capture of light by the event horizon. By studying this shadow scientists could measure the enormous mass of M87’s black hole. 
+
+### The Challenges 
+
+* **Scale**
+
+    The observations from Event Horizon Telescope (EHT) present challenges for existing data processing tools, arising from the rapid atmospheric phase fluctuations, wide recording bandwidth, and highly heterogeneous array.
+
+* **Calibration and Correlation**
+
+    Besides scheduling all of these coordinated observations of EHT, reducing the overall volume and complexity of data to aid analysis is a really hard problem to solve. To put things in perspective, EHT generates over 350 Terabytes worth of observed data per day, stored on high-performance helium filled hard drives.
+
+* **Source Imaging and Model Fitting**
+
+    The sequence of correlation and engineering releases represents a year-long effort of identifying and mitigating data issues, and developing new software and procedures that could reliably choose the most likely image based on actual measurements.
+
+{{< figure src="/images/content_images/cs/dataprocessbh.png" class="fig-center" caption="**EHT Data Processing Pipeline**" alt="data pipeline" align="middle" attr="(Diagram Credits: The Astrophysical Journal, Event Horizon Telescope Collaboration)" attrlink="https://iopscience.iop.org/article/10.3847/2041-8213/ab0c57" >}}
```

Comment:
I think this will work:

```
<div style="text-align:center">
<img src="./images/content_images/cs/dataprocessbh.png" />

<b> EHT Data Processing Pipeline </b>
</div>

```

Reply:
You can also try 

```
<p align="center">
  <img  src="./images/content_images/cs/dataprocessbh.png">
 <b> EHT Data Processing Pipeline </b>
</p>

```

Reply:
> You can also try
> 
> ```
> <p align="center">
>   <img  src="./images/content_images/cs/dataprocessbh.png">
>  <b> EHT Data Processing Pipeline </b>
> </p>
> ```

How do we use html to align left or centre say  inside {{ < figure ... >}} shortcode call?  If I add it in figure shortcode html then all calls to figure will be entered.  Not sure if that is a good idea?

Reply:
I tried this:  The text (sample text just for demo) is in centre but the image is still on LHS

<p align="center" align-items="center">
This is in center
{{< figure src="/images/content_images/cs/dataprocessbh.png" class="fig-center" caption="**EHT Data Processing Pipeline**" alt="data pipeline" align="middle" attr="(Diagram Credits: The Astrophysical Journal, Event Horizon Telescope Collaboration)" attrlink="https://iopscience.iop.org/article/10.3847/2041-8213/ab0c57" >}}
</p>
<img width="762" alt="Screenshot 2020-02-13 at 10 18 58 PM" src="https://user-images.githubusercontent.com/5890484/74457860-d7cc7c00-4eae-11ea-81a9-b3ccc3dd7444.png">

Reply:
Phew! Its finally done.  I simply had to NOT reuse figcaption css but create an identical style called csfigcaption and viola, image is in centre.  think it is something to do with the fact that this is the only image that shows up right after a list (text bulleted list).

Reply:
<img width="798" alt="Screenshot 2020-02-13 at 10 49 15 PM" src="https://user-images.githubusercontent.com/5890484/74460904-b15d0f80-4eb3-11ea-8c76-ea21e733015e.png">
<img width="555" alt="Screenshot 2020-02-13 at 10 49 58 PM" src="https://user-images.githubusercontent.com/5890484/74460963-c9cd2a00-4eb3-11ea-86f6-f439145890fa.png">
<img width="161" alt="Screenshot 2020-02-13 at 10 54 47 PM" src="https://user-images.githubusercontent.com/5890484/74461009-d94c7300-4eb3-11ea-8f03-919c7ef465b7.png">